app/main/startup.ts

```diff
@@ -11,21 +11,23 @@ export const setAutoLaunch = (AutoLaunchValue: boolean): void => {
 		return;
 	}
 
-	// On Mac, work around a bug in auto-launch where it opens a Terminal window
-	// See https://github.com/Teamwork/node-auto-launch/issues/28#issuecomment-222194437
-
-	const appPath = process.platform === 'darwin' ? app.getPath('exe').replace(/\.app\/Content.*/, '.app') : undefined; // Use the default
-
-	const ZulipAutoLauncher = new AutoLaunch({
-		name: 'Zulip',
-		path: appPath,
-		isHidden: false
-	});
 	const autoLaunchOption = ConfigUtil.getConfigItem('startAtLogin', AutoLaunchValue);
 
-	if (autoLaunchOption) {
-		ZulipAutoLauncher.enable();
+	// setLoginItemSettings doesn't support linux
+	if (process.platform === 'linux') {
+		const ZulipAutoLauncher = new AutoLaunch({
+			name: 'Zulip',
```

Comment:
@vsvipul you removed the path arg here.

Reply:
@akashnimare  We were changing the path only for mac previously and using the default for linux. As we now use AutoLaunch for only linux, this is no longer required. See https://www.npmjs.com/package/auto-launch .
```
For NW.js and Electron apps, you don't have to specify the path. We guess based on process.execPath.
```

Reply:
Got it. But do we need to update the auto-launch version?

Reply:
We are already on the latest version i.e. 5.0.5 .